access/templates/access/graded_form.html

```diff
@@ -1,32 +1,39 @@
 {% load i18n %}
-<form method="post" action="{{ post_url }}"{% if result.form.multipart %} enctype="multipart/form-data"{% endif %}>
+{% load access %}
+<form method="post" action="{{ post_url }}"{% if result.form.multipart %} enctype="multipart/form-data"{% endif %} class="aplus-quiz1">
 	{% for field in result.form %}
+
 	{% if field.field.open_set %}
-	<fieldset id="{{ field.field.open_set }}"{% if result.accepted and field.field.group_errors %} class="alert {% if field.field.open_set in result.error_groups %}alert-danger{% else %}alert-success{% endif %}"{% endif %}>
+	<!-- Beginning of all questions -->
+		<fieldset id="{{ field.field.open_set }}"{% if result.accepted and field.field.group_errors %} class="alert {% if field.field.open_set in result.error_groups %}alert-danger{% else %}alert-success{% endif %}"{% endif %}>
 		{% if field.field.set_title %}<legend>{{ field.field.set_title }}</legend>{% endif %}
 	{% endif %}
 
 		{% if field.field.type == "static" %}
-		{{ field.field.more|safe }}
+		<div class="question-description">
+			{{ field.field.more|safe }}
+		</div>
 		{% elif field.field.type == "hidden" %}
 			{{ field }}
 		{% else %}
 
 		{% if not field.field.row_label or field.field.open_table %}
-		<div class="{{ field.field.html_class }} {% if field.errors %} has-error{% elif not field.field.group_errors and field.field.name in result.error_fields %} has-error{% endif %}">
+		<!-- Individual question heading (individual form-group) -->
+		<div class="question-area {{ field.field.html_class }} {% if field.errors %} has-error{% endif %}">
 
 			{% if field.field.required %}<span class="required">*</span>{% endif %}
 
 			{% if field.label %}
-			{% if field.field.choice_list %}
-			<label>{{ field.label }}</label>
-			{% else %}
-			{{ field.label_tag }}
-			{% endif %}
+				{% if field.field.choice_list %}
+					<label class="control-label">{{ field.label }}</label>
+				{% else %}
+					<label class="control-label">{{ field.label_tag }}</label>
+				{% endif %}
 			{% endif %}
 
 			{% if not field.field.group_errors %}
 			{% if result.accepted %}
+			<!-- Points badge -->
 				{% if not exercise.feedback or field.field.points > 0 %}
 				<span class="badge badge-{% if field.field.grade_points > 0 and field.field.grade_points < field.field.max_points %}warning{% elif field.field.name in result.error_fields %}danger{% else %}success{% endif %}">
```

Comment:
Can you make the same thing for the badges as you did for the glyphicons in a later PR?

Reply:
Yes, sure! I will create a pull request this week.